flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliStrings.java

```diff
@@ -73,54 +58,41 @@ private CliStrings() {
-                    .append(
-                            formatCommand(
-                                    "USE",
-                                    "Sets the current default database. Experimental! Syntax: \"USE <name>;\""))
+                                    "EXPLAIN",
+                                    "Describes the execution plan of a query or table with the given name."))
                     .append(
                             formatCommand(
-                                    "LOAD MODULE",
-                                    "Load a module. Syntax: \"LOAD MODULE <name> [WITH ('<key1>' = "
-                                            + "'<value1>' [, '<key2>' = '<value2>', ...])];\""))
+                                    "BEGIN STATEMENT SET",
+                                    "Begins a statement set. Syntax: \"BEGIN STATEMENT SET;\""))
+                    .append(formatCommand("END", "Ends a statement set. Syntax: \"END;\""))
+
                     .append(
                             formatCommand(
-                                    "UNLOAD MODULE",
-                                    "Unload a module. Syntax: \"UNLOAD MODULE <name>;\""))
+                                    "ADD JAR",
+                                    "Adds the specified jar file to the submitted jobs' classloader. Syntax: \"ADD JAR '<path_to_filename>.jar'\""))
                     .append(
                             formatCommand(
-                                    "USE MODULES",
-                                    "Enable loaded modules. Syntax: \"USE MODULES <name1> [, <name2>, ...];\""))
+                                    "REMOVE JAR",
+                                    "Removes the specified jar file from the submitted jobs' classloader. Syntax: \"REMOVE JAR '<path_to_filename>.jar'\""))
                     .append(
                             formatCommand(
-                                    "BEGIN STATEMENT SET",
-                                    "Begins a statement set. Syntax: \"BEGIN STATEMENT SET;\""))
-                    .append(formatCommand("END", "Ends a statement set. Syntax: \"END;\""))
+                                    "SHOW JARS",
+                                    "Shows the list of user-specified jar dependencies. This list is impacted by the --jar and --library startup options as well as the ADD/REMOVE JAR commands."))
                     .style(AttributedStyle.DEFAULT.underline())
                     .append("\nHint")
                     .style(AttributedStyle.DEFAULT)
                     .append(
                             ": Make sure that a statement ends with \";\" for finalizing (multi-line) statements.")
+                    // About Documentation Link.
+                    .style(AttributedStyle.DEFAULT)
+                    .append(
+                            "\nPlease visit https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/table/sql/overview/ for more details.")
```

Comment:
Hi,
I know I'm a bit nit-picking here, sorry, although how about giving some more indication that what the use will find in that page is SQL command they can type? 
Maybe something like:
```
.append("You can also type any Flink SQL statement, please visit https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/table/sql/overview/ for more details.")
```

Reply:
That looks good.

Reply:
@sv3ndk Thanks you for your suggestions. And I made some minor changes. I’d appreciate it if you would help me to review it .

Reply:
Hi @RocMarshal Nice PR, as far as I'm concerned it looks good, thanks for having worked on that :+1: 
I'm just another Flink user who sometimes sends minor contributions, I can't approve nor merge anything.
Other people in this thread are actual Flink members, I'm sure they received the notifications => I'm assuming a reaction will be posted below.